src/seedu/addressbook/parser/Parser.java

```diff
@@ -97,12 +84,17 @@ public Command parseCommand(String userInput) {
         case ExitCommand.COMMAND_WORD:
             return new ExitCommand();
 
+        case RandomCommand.COMMAND_WORD:
+             return new RandomCommand();
```

Comment:
/nit inconsistant spacing. Convention for this module is 4 space.

Reply:
Thanks for the review. I will take note of that!